imap_processing/codice/codice_l1a.py

```diff
@@ -27,16 +27,22 @@
 from imap_processing.cdf.utils import met_to_j2000ns
 from imap_processing.codice import constants
 from imap_processing.codice.codice_l0 import decom_packets
+from imap_processing.codice.decompress import decompress
 from imap_processing.codice.utils import CODICEAPID, add_metadata_to_array
 from imap_processing.utils import group_by_apid, sort_by_time
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-# TODO: Decom data arrays need to be decompressed
 # TODO: In decommutation, how to have a variable length data and then a checksum
 #       after it? (Might be fixed with new XTCE script updates)
 # TODO: Add support for decomming multiple APIDs from a single file
+# TODO: Add these as variables in CDF: SPIN_PERIOD, ST_BIAS_GAIN_MODE,
```

Comment:
I'm curious if you write tickets for all of your TODOs. I know that I should be better about forcing myself to do that.

Reply:
Historically I haven't been creating tickets for these, but I think I am going to start doing that, especially since we are now trying to have more incremental progress / smaller PRs / project boards tied to tickets.